docs/sphinx/_ext/sphinx-mdx-builder/sphinxcontrib/mdxbuilder/writers/mdx.py

```diff
@@ -234,9 +234,20 @@ def unknown_visit(self, node: Element) -> None:
         raise nodes.SkipNode
 
     def visit_Text(self, node: nodes.Text) -> None:
+        # print("->", type(node.parent), node.parent)
         if isinstance(node.parent, nodes.reference):
             return
+
         content = node.astext()
+
+        # Skip render of messages from `dagster_sphinx` `inject_param_flag`
+        # https://github.com/dagster-io/dagster/blob/colton/inline-flags/docs/sphinx/_ext/dagster-sphinx/dagster_sphinx/docstring_flags.py#L36-L63
+        if "This parameter may break" in content:
+            return
+
+        if "This parameter will be removed" in content:
+            return
```

Comment:
@PedramNavid not proud of this, as when `dagster_sphinx/docstring_flags.py` is changed, this conditional may fail.

1st attempt was to add `self.is_inline_flags += 1` to track state, however, this text block occurs _after_ the `depart_inline_flags`. So it would be decremented back to 0.

2nd attempt was to look at the previous state in `self.states`, however, that results in less intuitive code like `if 'flag flag-` in `self.states[-1][-3][-1]`.

Reply:
sad but works so happy